repo-agent/review-ui/review-api/main.go

```diff
@@ -1155,3 +1166,26 @@ func deleteIssue(c *gin.Context) {
 
 	c.Status(http.StatusOK)
 }
+
+func proxy(c *gin.Context) {
+	proxyURL := c.Query("url")
+	if proxyURL == "" {
+		c.JSON(http.StatusBadRequest, gin.H{"error": "url query parameter is required"})
+		return
+	}
+
+	resp, err := http.Get(proxyURL)
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, gin.H{"error": fmt.Sprintf("Failed to fetch url: %v", err)})
+		return
+	}
+	defer resp.Body.Close()
+
+	body, err := io.ReadAll(resp.Body)
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, gin.H{"error": fmt.Sprintf("Failed to read response body: %v", err)})
+		return
+	}
+
+	c.String(http.StatusOK, string(body))
```

Comment:
Do we want this to be http.StatusOk or the status code that the /proxy response got.

eg:
```go
c.String(resp.StatusCode, string(body))
```